sos/plugins/etcd.py

```diff
@@ -23,15 +23,46 @@ class etcd(Plugin, RedHatPlugin):
     """etcd plugin
     """
 
+    packages = ('etcd',)
+
     def setup(self):
-        self.add_copy_spec("/etc/etcd")
+        self.address = self.get_address()
+        self.cmd = 'etcdctl '
+
+        self.add_copy_spec('/etc/etcd')
+
+        subcmds = ['--version',
+                   'member list',
+                   'cluster-health',
+                   'ls --recursive',
+                   ]
+
+        for sub in subcmds:
+            self.add_cmd_output("%s %s" % (self.cmd, sub))
+
```

Comment:
Ack. I did the for loop since that's what I used in plugins like `docker`. Fixed.